i18n/de.po

```diff
@@ -18,446 +18,1140 @@ msgstr ""
-#: templates/index.html:32 templates/index.html:44
-msgid "like"
-msgstr ""
+#: templates/features.html:20
+msgid " of a polynomial "
+msgstr " eines Polynoms "
 
-#: templates/index.html:26
-msgid "expansion"
-msgstr ""
+#: templates/features.html:21
+msgid "Functions:"
+msgstr "Funktionen:"
 
-#: templates/index.html:27
-msgid "functions"
+#: templates/features.html:22
+#, python-format
+msgid ""
+"\n"
+"                    <a href=\"%(trig)s\">trigonometric</a>,\n"
+"                    <a href=\"%(hyperbolic)s\">hyperbolic</a>,\n"
+"                    <a href=\"%(exponential)s\">exponential</a>,\n"
+"                    <a href=\"%(root)s\">roots</a>,\n"
+"                    <a href=\"%(logarithm)s\">logarithms</a>,\n"
+"                    <a href=\"%(abs)s\">absolute value</a>,\n"
+"                    <a href=\"%(spherical)s\">spherical harmonics</a>,\n"
+"                    <a href=\"%(factorial)s\">factorials and gamma "
+"functions</a>,\n"
+"                    <a href=\"%(zeta)s\">zeta functions</a>,\n"
+"                    <a href=\"%(polynomial)s\">polynomials</a>,\n"
+"                    <a href=\"%(special)s\">special functions</a>, ..."
+msgstr ""
+
+#: templates/features.html:46
+msgid "Substitution:"
+msgstr "Ersetzung:"
+
+#: templates/features.html:46
+msgid "example"
+msgstr "Beispiel"
+
+#: templates/features.html:47
+msgid "Numbers:"
+msgstr "Zahlen:"
+
+#: templates/features.html:48
+#, python-format
+msgid ""
+"\n"
+"                    <a href=\"%(integer)s\">arbitrary precision integers</"
+"a>,\n"
+"                    <a href=\"%(rational)s\">rationals</a>,\n"
+"                    <a href=\"%(float)s\">and floats</a>\n"
+"                    "
 msgstr ""
 
-#: templates/index.html:28
-msgid "complex numbers"
-msgstr ""
+#: templates/features.html:58
+msgid "Noncommutative symbols"
+msgstr "Nichtkommutative Symbole"
 
-#: templates/index.html:30
-msgid "differentiation"
-msgstr ""
+#: templates/features.html:59
+msgid "Pattern matching"
+msgstr "Muster-Erkennung"
 
-#: templates/index.html:31
-msgid "taylor (laurent) series"
-msgstr ""
+#: templates/features.html:65
+msgid "Polynomials"
+msgstr "Polynome"
 
-#: templates/index.html:32
-msgid "substitution"
-msgstr ""
+#: templates/features.html:69
+msgid " division, gcd, ... "
+msgstr " Division, GCD, ... "
 
-#: templates/index.html:32
-msgid "or"
-msgstr ""
+#: templates/features.html:70
+msgid "Factorization"
+msgstr "Faktorisierung"
 
-#: templates/index.html:34
-msgid "arbitrary precision integers, rationals and floats"
-msgstr ""
+#: templates/features.html:71
+msgid "Square-free decomposition"
+msgstr "Quadratfreie Zerlegung"
 
-#: templates/index.html:35
-msgid "noncommutative symbols"
-msgstr ""
+#: templates/features.html:72
+msgid "Gröbner bases"
+msgstr "Gröbnerbasen"
+
+#: templates/features.html:73
+msgid "Partial fraction decomposition"
+msgstr "Teilbruchzerlegung"
+
+#: templates/features.html:74
+msgid "Resultants"
+msgstr "Resultate"
```

Comment:
Is this correct?

Reply:
Good catch
```suggestion
msgstr "Ergebnisse"
```

Reply:
This one is meant: https://de.wikipedia.org/wiki/Resultante, correct?

Reply:
> correct?

Yes.